src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -178,122 +198,78 @@ void GeantPhysicsList::add_e_processes()
+        model->SetActivationLowEnergyLimit(msc_energy_limit);
+        process->SetEmModel(model.release());
+        physics_list->RegisterProcess(process.release(), p);
 
         CELER_LOG(debug) << "Loaded Coulomb scattering with "
                             "G4eCoulombScatteringModel";
     }
 
-    if (true)
+    if (options_.msc != MscModelSelection::none)
     {
         // Multiple scattering: Urban (low E) and WentzelVI (high E) models
         double msc_energy_limit = G4EmParameters::Instance()->MscEnergyLimit();
-        auto   msc_electron     = std::make_unique<G4eMultipleScattering>();
-        auto   msc_positron     = std::make_unique<G4eMultipleScattering>();
 
-        if (true)
+        auto process = std::make_unique<G4eMultipleScattering>();
+
+        if (options_.msc == MscModelSelection::urban)
         {
-            // Urban model
-            // Electron
-            auto urban_msc_electron = std::make_unique<G4UrbanMscModel>();
-            urban_msc_electron->SetHighEnergyLimit(msc_energy_limit);
-            msc_electron->SetEmModel(urban_msc_electron.release());
-
-            // Positron
-            auto urban_msc_positron = std::make_unique<G4UrbanMscModel>();
-            urban_msc_positron->SetHighEnergyLimit(msc_energy_limit);
-            msc_positron->SetEmModel(urban_msc_positron.release());
-
-            CELER_LOG(debug) << "Loaded multiple scattering with "
-                                "G4UrbanMscModel";
+            auto model = std::make_unique<G4UrbanMscModel>();
+            model->SetHighEnergyLimit(msc_energy_limit);
+            process->SetEmModel(model.release());
         }
 
-        // DISABLED
-        if (false)
+        if (options_.msc == MscModelSelection::wentzel_vi)
         {
-            // WentzelVI model
-            // Electron
-            auto wentzelvi_msc_electron = std::make_unique<G4WentzelVIModel>();
-            wentzelvi_msc_electron->SetLowEnergyLimit(msc_energy_limit);
-            msc_electron->SetEmModel(wentzelvi_msc_electron.release());
-
-            // Positron
-            auto wentzelvi_msc_positron = std::make_unique<G4WentzelVIModel>();
-            wentzelvi_msc_positron->SetLowEnergyLimit(msc_energy_limit);
-            msc_positron->SetEmModel(wentzelvi_msc_positron.release());
-
-            CELER_LOG(debug) << "Loaded multiple scattering with "
-                                "G4WentzelVIModel";
+            auto model = std::make_unique<G4WentzelVIModel>();
+            model->SetHighEnergyLimit(msc_energy_limit);
+            process->SetEmModel(model.release());
         }
 
-        physics_list->RegisterProcess(msc_electron.release(), electron);
-        physics_list->RegisterProcess(msc_positron.release(), positron);
+        physics_list->RegisterProcess(process.release(), p);
+
+        CELER_LOG(debug) << "Loaded multiple scattering with "
+                            "G4UrbanMscModel";
```

Comment:
Given that we have both `MscModelSelection` set up already, this debug msg can be misleading. I suggest adding a specific message for each msc model loaded.

Reply:
Good point.